ui/widget.js

```diff
@@ -514,6 +514,16 @@ $.Widget.prototype = {
 			}
 		}
 
+		this._on( options.element, {
+			"remove": function( event ) {
+				$.each( that.classesElementLookup, function( key, value ) {
+					if ( $.inArray( event.target ) ) {
```

Comment:
@scottgonzalez @mgol Looks like doing the `inArray` check first is MUCH faster
http://jsbin.com/zaparohuba/edit?html,js,output